docs/app/Examples/collections/Grid/Variations/GridTextAlignmentExample.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Grid, Menu } from 'stardust'
+
+// TODO: Update <Menu> usage after v1 API
+
+const { Column, Row } = Grid
+
+const GridTextAlignmentExample = () => (
+  <Grid textAlign='center' columns={3}>
```

Comment:
I believe this file should be `GridTextCenterAlignmentExample` (added Center) to match the others, yea?